jdbc/src/main/resources/interpreter-setting.json

```diff
@@ -143,6 +143,13 @@
         "defaultValue": "1000",
         "description": "Query interval for hive statement",
         "type": "number"
+      },
+      "zeppelin.jdbc.hive.tagged.engines.exclude": {
```

Comment:
I think this could be a boolean property, it is not necessary to differentiate mr or tez, because the user can switch engine easily.

Reply:
Get it ! 😀

Reply:
Done. 
By the way I fixed a problem about the type of  default value of   `"zeppelin.jdbc.auth.kerberos.proxy.enable"`.
If it is string `"true"`  instead of  boolean `true`,  the checkbox in UI would not ✅ .

Reply:
Also need to update `jdbc.md` for this new property.

Reply:
> Also need to update `jdbc.md` for this new property.

Hi @zjffdu , This new property is only related to hive, should I put it into `hive.md` ?   
I didn't find a good place for this property in `jdbc.md`.

Reply:
Both places should update. There's one section about hive in `jdbc.md`, https://zeppelin.apache.org/docs/0.10.1/interpreter/jdbc.html#apache-hive. BTW, it looks like to we need to converge the hive related doc in these 2 places. But it could be done in another PR.

Reply:
Hi @zjffdu , I have put all hive-related properties into `hive.md` and add a link to `hive.md` in `jdbc.md`.